### PR TITLE
fix(docs): grammar corrections - I184

### PR DESCRIPTION
**Pull request by whitehawk0011**
# Issue #<NUMBER HERE>
#184 
https://github.com/accordproject/techdocs/issues/184
<OVERALL SUMMARY OF PULL REQUEST>

### Changes
- <INSERT DESCRIPTION OF CHANGES>
changed just and simply in sentences
  - <SUB NOTES OF DESCRIPTIONS>
- <SUMMARIZE ALL COMMITS IN PULL REQUEST>

### Flags
- <DESCRIBE ISSUES OR HOLDS FOR REVIEWERS>

### Related Issues
- Issue #<NUMBER HERE>
- Pull Request #<NUMBER HERE>

### CODE REVIEW

File: docs/accordproject.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ The following screenshot shows the text of an acceptance of delivery clause.
 
 ![Example Text](/docs/assets/grammar.png)
 
-The text is written in plain English with variables between `{{` and `}}` and highlighted in blue. Using variables is useful so the same template can be used in different agreements by simply replacing them by different values.
+The text is written in plain English with variables between `{{` and `}}` and highlighted in blue. Using variables is useful so the same template can be used in different agreements by  replacing them by different values.
```
Comment (irmerk) [inline docs/accordproject.md]: There is a space left here.

File: docs/accordproject-hello.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ Cicero archives are files with a `.cta` extension, which includes all the differ
 
 ## Parse a Valid Clause Text
 
-Using your terminal `cd` into the directory that contains the template archive you just downloaded, then create a sample clause text `sample.txt` which contains the following text:
+Using your terminal `cd` into the directory that contains the template archive you have downloaded, then create a sample clause text `sample.txt` which contains the following text:
```
Comment (irmerk) [inline docs/accordproject-hello.md]: This change can be reverted.

**Comment by whitehawk0011** [comment]
ok , soryy for the issues, I will commit them shortly

**Comment by whitehawk0011** [comment]
done !

**Comment by irmerk** [comment]
Closed for #219

**Pull request closed without merging**